includes/class-freemius.php

```diff
@@ -10634,6 +10678,18 @@ function is_tracking_allowed() {
             return ( is_object( $this->_site ) && $this->_site->is_tracking_allowed() );
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since  2.3.3
+         *
+         * @return bool
+         */
+        function is_bundle_license_auto_activation_enabled() {
+            return $this->is_addon() ?
+                $this->_parent->is_bundle_license_auto_activation_enabled() :

```

Comment:
@fajardoleo I'd recommend to tweak it to:
`( is_object( $this->_parent ) && $this->_parent->is_bundle_license_auto_activation_enabled() )`

Just in case that this method is called from an add-on before the parent instance was initialized.